LoRaEngine/modules/LoRaWanNetworkSrvModule/Logger/LoggerConfiguration.cs

```diff
@@ -16,8 +16,8 @@ public class LoggerConfiguration
         public bool LogToConsole { get; set; } = true;
 
         // Gets/sets the logging level
-        // Default: 0 (Always logging)
-        public int LogLevel { get; set; } = 0;
+        // Default: 4 (Error)
```

Comment:
I would change this from int to LogLevel, avoiding casts down the road and making the code cleaner.
The downside is it will break all checks for logLevel > (int)value. To address it I would implement Logger.IsEnabled(LogLevel) to keep a similar behavior with the ILogger interface.

It will enable a future change to a 3rd-party logging library smoothly.

Reply:
Done! No problem with the checks as Microsoft.Extensions.Logging.LogLevel can be cast to int.